deploy/k8s/standalone-agent.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     k8s-app: elastic-agent
 data:
-  agent.yml: |-
```

Comment:
perhaps better to stay  with *.yml for the agent config since it's the beats convention to use <beatname>.yml for the beat's config file

Reply:
so maybe we should stick with `yml` across the entire lib?

Reply:
That would work.